src/wallet/walletutil.h

```diff
@@ -11,4 +11,7 @@
 //! Get the path of the wallet directory.
 fs::path GetWalletDir();
 
+//! Get wallets in wallet directory.
+std::vector<fs::path> ListWalletDir();
+
```

Comment:
Nit: Add some trivial test for `ListWalletDir` to make it technically in use.

Reply:
There is `listwaletdir` RPC, which has some tests.